doc/man1/enc.pod

```diff
@@ -103,6 +104,12 @@ the B<-pass> argument.
 Use the specified digest to create the key from the passphrase.
 The default algorithm is sha-256.
 
+=item B<-iter count>
+
+Use a given number of iterations on the password in deriving the encryption key.
+High values increase the time required to brute-force the resulting file.
+This option enables the use of PBKDF2 algorithm to derive the key.
```

Comment:
I understand this is no longer require since you agree with Matt's proposal to generate IV from password.